1.0/index.md

```diff
@@ -216,7 +216,7 @@ title: Welcome
     </firebase-collection>
     <template is="dom-repeat" items="{{data}}">
       <contact-card starred="{{item.starred}}">
-        <img src="{{item.img}}" alt="{{item.name}}">
+        <img src="{{item.img}}">
```

Comment:
Per the original bug, this alt text duplicates the name that immediately follows the image, meaning the screen reader reads it as "Image <person name> <person name>".